collectors/node/cpu.go

```diff
@@ -167,3 +167,11 @@ func round(f float64) float64 {
 	shift := math.Pow(10, float64(2))
 	return math.Floor(f*shift+.5) / shift
 }
+
+// Helper function: replace all negative numbers with zero
+func gtZero(n float64) float64 {
+	if math.Signbit(n) {
```

Comment:
maybe it makes sense to move `n < 0` to helper function `round`? i don't think you ever need to round negative numbers?